content/cli/v11/configuring-npm/package-json.mdx

```diff
@@ -305,6 +305,39 @@ These can not be included.
 
 The "exports" provides a modern alternative to "main" allowing multiple entry points to be defined, conditional entry resolution support between environments, and preventing any other entry points besides those defined in "exports". This encapsulation allows module authors to clearly define the public interface for their package. For more details see the [node.js documentation on package entry points](https://nodejs.org/api/packages.html#package-entry-points)
 
+### type
+
+The `type` field defines the module format that Node.js should use for `.js` files in the package. When set to `"module"`, Node.js will treat `.js` files as ES modules. When set to `"commonjs"` (or when the field is omitted), Node.js will treat `.js` files as CommonJS modules.
```

Comment:
i think rather than all of this content, it should be nothing more than a link to https://nodejs.org/api/packages.html#type, so that there's not two sources of truth to keep up to date.

Reply:
oh sure, that works fine :-) just, nothing technical, ie that would need to be updated in the future.